text-utils/ul.c

```diff
@@ -254,12 +239,18 @@ static void sig_handler(int signo __attribute__((__unused__)))
 	_exit(EXIT_SUCCESS);
 }
 
+static int ul_putwchar(int c)
+{
+	if (putwchar(c) == WEOF)
+		return EOF;
```

Comment:
That's right. Note that the fallbacks are in include/widechar.h. For example `# define putwchar putchar`.

Reply:
Nice, I had forgot `include/widechar.h` even exists.